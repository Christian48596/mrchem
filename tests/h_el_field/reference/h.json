{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            1.0
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            1.0
          ]
        }
      ],
      "multiplicity": 2
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "h.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "external_operator": {
          "electric_field": [
            0.0,
            0.0,
            0.01
          ],
          "r_O": [
            0.0,
            0.0,
            0.0
          ]
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "Hartree-Fock",
        "prec": 0.001,
        "restricted": false,
        "screen": 12.0,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.001,
        "helmholtz_prec": -1.0,
        "kain": 3,
        "light_speed": -1.0,
        "localize": false,
        "max_iter": 5,
        "method": "Hartree-Fock",
        "orbital_thrs": 0.01,
        "proj_prec": 0.001,
        "rotation": 0,
        "shared_memory": false,
        "smooth_prec": 0.001,
        "start_prec": 0.001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        1.0
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.03433749346671411,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            0.03433749346671411
          ],
          "vector_el": [
            0.0,
            0.0,
            -0.9656625065331544
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.9999999999998685
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            1.0
          ]
        }
      ],
      "multiplicity": 2,
      "orbital_energies": {
        "energy": [
          -0.49122851788137456
        ],
        "occupation": [
          1.0
        ],
        "spin": [
          "a"
        ],
        "sum_occupied": -0.49122851788137456
      },
      "scf_energy": {
        "E_ee": 0.31200384844043044,
        "E_eext": 0.009656625065331532,
        "E_el": -0.49069714945162485,
        "E_en": -1.0008375505974554,
        "E_kin": 0.5010151445102491,
        "E_next": -0.009999999999998685,
        "E_nn": 0.0,
        "E_nuc": -0.009999999999998685,
        "E_tot": -0.5006971494516236,
        "E_x": -0.31253521687018054,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 0.3114941166208955,
        "E_eext": 0.0100000159361738,
        "E_el": -0.4904569092049826,
        "E_en": -1.0000031136308007,
        "E_kin": 0.5000385924124847,
        "E_next": -0.009999999999998685,
        "E_nn": 0.0,
        "E_nuc": -0.009999999999998685,
        "E_tot": -0.5004569092049813,
        "E_x": -0.31198652054373577,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 0.3119005310442684,
              "E_eext": 0.009765328387436557,
              "E_el": -0.4906594071008387,
              "E_en": -1.000651465674207,
              "E_kin": 0.5007536983345298,
              "E_next": -0.009999999999998685,
              "E_nn": 0.0,
              "E_nuc": -0.009999999999998685,
              "E_tot": -0.5006594071008375,
              "E_x": -0.3124274991928665,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -0.5006594071008375,
            "energy_update": 0.0002024978958561352,
            "mo_residual": 0.012605790029833901,
            "wall_time": 2.173039446
          },
          {
            "energy_terms": {
              "E_ee": 0.31200384844043044,
              "E_eext": 0.009656625065331532,
              "E_el": -0.49069714945162485,
              "E_en": -1.0008375505974554,
              "E_kin": 0.5010151445102491,
              "E_next": -0.009999999999998685,
              "E_nn": 0.0,
              "E_nuc": -0.009999999999998685,
              "E_tot": -0.5006971494516236,
              "E_x": -0.31253521687018054,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -0.5006971494516236,
            "energy_update": 3.774235078612964e-05,
            "mo_residual": 0.005628388907140271,
            "wall_time": 1.589985264
          }
        ],
        "wall_time": 3.763115639
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
