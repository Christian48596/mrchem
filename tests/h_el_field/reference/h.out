                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         rebase-zora                              ***
***         Git commit hash    faecb83984c1d010ec90-dirty               ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Thu Mar 10 17:22:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        1.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        1.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : False
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               0
 Total electrons         :                                               1
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : Hartree-Fock
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.500038592412
 E-N energy              :            (au)                 -1.000003113631
 Coulomb energy          :            (au)                  0.311494116621
 Exchange energy         :            (au)                 -0.311986520544
 X-C energy              :            (au)                  0.000000000000
 Ext. field (el)         :            (au)                  0.010000015936
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.000000000000
 Ext. field (nuc)        :            (au)                 -0.010000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -0.490456909205
 Nuclear energy          :            (au)                 -0.010000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.004569092050e-01
                         :      (kcal/mol)             -3.140414493526e+02
                         :        (kJ/mol)             -1.313949424944e+03
                         :            (eV)             -1.361812565731e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -0.490949313128
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.490949313128
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : Hartree-Fock
 Checkpointing           : Off
 Max iterations          : 5
 KAIN solver             : 3
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-02
 ZORA potential          : None
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00          -0.500456909205         -5.004569e-01
    1          1.260579e-02          -0.500659407101         -2.024979e-04
    2          5.628389e-03          -0.500697149452         -3.774235e-05
---------------------------------------------------------------------------
                      SCF converged in 2 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        1.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        1.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.501015144510
 E-N energy              :            (au)                 -1.000837550597
 Coulomb energy          :            (au)                  0.312003848440
 Exchange energy         :            (au)                 -0.312535216870
 X-C energy              :            (au)                  0.000000000000
 Ext. field (el)         :            (au)                  0.009656625065
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.000000000000
 Ext. field (nuc)        :            (au)                 -0.010000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -0.490697149452
 Nuclear energy          :            (au)                 -0.010000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.006971494516e-01
                         :      (kcal/mol)             -3.141922023822e+02
                         :        (kJ/mol)             -1.314580175620e+03
                         :            (eV)             -1.362466292717e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                 -0.491228517881
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.491228517881
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000       -0.965663
 Magnitude               :            (au)                        0.965663
                         :         (Debye)                        2.454469
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.000000
 Magnitude               :            (au)                        1.000000
                         :         (Debye)                        2.541746
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.034337
 Magnitude               :            (au)                        0.034337
                         :         (Debye)                        0.087277
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  4s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
